doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_application.rst

```diff
@@ -0,0 +1,863 @@
+.. _migration_cs3_to_2_6_99_cs2_app:
+
+Migrate your application to |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#####################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes the changes you should be aware of when migrating your application from |NCS| for the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.6.99-cs2.
+
+Required changes
+****************
+
+The following changes are mandatory to make your application work in the same way as in previous releases.
+
+This section describes the changes related to samples and applications.
+
+Samples and applications
+========================
+
+General
+-------
+
+.. toggle::
+
+  * The nRF54H20 DK samples and applications are now using the following board names:
+
+    * Application core: ``nrf54h20dk_nrf54h20_cpuapp``
+    * Radio core: ``nrf54h20dk_nrf54h20_cpurad``
+    * PPR core: ``nrf54h20dk_nrf54h20_cpuppr``
```

Comment:
Shouldn't these be of the format nrf54h20dk/nrf54h20/cpuapp, @gmarull ?

Reply:
It is correct. This cs release will not have hardware model v2.

I suggest to give the previous names of boards - it would be easier too do a lookup